tests/test_main.py

```diff
@@ -365,6 +365,25 @@ class Config:
     assert '"TestModel" object has no field "b"' in str(exc_info)
 
 
+class ConstModel(BaseModel):
+    a: int = Schema(3, const=True)
+
+
+def test_const_validates():
+    m = ConstModel(a=3)
+    assert m.a == 3
+
+
+def test_const_is_required():
+    with pytest.raises(ValidationError):
```

Comment:
I was umming and ahhing over whether to raise or use the default value. My thinking was that it would be a bit weird to have a constant as part of your schema without requiring it to be there, _however_ the JSON Schema spec doesn't specify that it needs to be required so I'm happy to change it.

Reply:
Thanks, I understand the intention.

Let's see if @samuelcolvin has a different opinion on that.